certbot-apache/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
 acme[dev]==0.29.0
-certbot[dev]==0.39.0
+-e .[dev]
```

Comment:
Because otherwise you get transitively the old requirement for pylint which is in the `dev` extra of certbot `setup.py`, and that is then pinned to an incompatible version for Python 2.7.

Using the current version of certbot, the pylint requirement is moved to `dev3` extra, an so is ignored in oldest tests.

Reply:
Updated `setup.py`